docker/Dockerfile.local

```diff
@@ -9,6 +9,7 @@ RUN apt-get install -y python3-pip git wget psmisc
 RUN apt-get install -y cmake
 
 # Install Pytorch
+RUN pip3 install networkx==3.1
```

Comment:
Which component needs this?

Reply:
Torch needs this. Currently the version torch automatically download is only compatible with python 3.9 while the base image only have 3.8.